File: trunk/quote/Projects/SampleProperties/ComputationProperiesFactory.cs
using System;

using Host;
using Model.Template.Ext;

namespace SampleProperties 
{
    [Register(Key = typeof(Model.Template.Ext.IComputationPropertiesFactory))]
    public class ComputationProperiesFactory : IComputationPropertiesFactory
    {

        public Model.Common.SavableProperties CreateComputationProperties(
            Model.Template.Header header, int id)
        {
            return new DisplayableComputationProperties(new SampleComputationProperties(header));
        }

    }
}

File: trunk/quote/Projects/DekalbProperties/ComputationProperiesFactory.vb
Imports Host

<Register(Key:=GetType(Model.Template.Ext.IComputationPropertiesFactory))>
Public Class ComputationPropertiesFactory
    Implements Model.Template.Ext.IComputationPropertiesFactory

    Public Function CreateComputationProperties(header As Model.Template.Header, _
                                                id As Integer) _
                                            As Model.Common.SaveableProperties _
                                            Implements Model.Template.Ext.IComputationPropertiesFactory.CreateComputationProperties
        Return New DisplayableComputationProperties(New DekalbComputationProperties(header))
    End Function

End Class
